handler.go

```diff
@@ -875,6 +882,20 @@ func (h *Handler) readURLQueryRequest(r *http.Request) (*QueryRequest, error) {
 	}, nil
 }
 
+func (h *Handler) getValidURLQuery(r *http.Request) map[string]bool {
+	validQuery := make(map[string]bool)
+	args := reflect.ValueOf(QueryRequest{})
```

Comment:
Small nitpick: since `args.Type()` is used in a loop below, I think it would be better to save it in a var like `argsType` instead of `args`.